content/en/tracing/custom_instrumentation/dotnet.md

```diff
@@ -26,7 +26,7 @@ This page details common use cases for adding and customizing observability with
 
 Add the `Datadog.Trace` [NuGet package][1] to your application. To create new spans, access the global tracer through the `Datadog.Trace.Tracer.Instance` property.
 
-Custom instrumentation is supported on **.NET Framework 4.5+** for Windows and on **.NET Core 2.1, 3.0, and 3.1** for Windows and Linux.
+Custom instrumentation is supported on **.NET Framework 4.5+** for Windows, on **.NET Core 2.0+** for Windows and Linux, and on **.NET 5** for Windows and Linux.
```

Comment:
This adds .NET Core 2.0 and 2.2 as supported versions, which we didn't have listed before. Was this intentional?

Reply:
Yes this was intentional on my part. I figured that manual instrumentation (consuming the SDK) is completely supported by .NET Standard 2.0 so we shouldn't shy away from declaring support for it. The one thing that has conditional support is automatic instrumentation. Do you agree with this thought process?

Reply:
Ok. My concern was that we don't actually run any tests on 2.0 or 2.2 If you feel confident about them, feel free to resolve my comment 😊